tests/integ/test_model_card.py

```diff
@@ -212,3 +376,130 @@ def test_model_card_create_read_update_and_delete(
 
         s3.get_object(Bucket=pdf_bucket, Key=pdf_key)
         assert "The specified key does not exist" in str(exception.value)
+
+
+@pytest.mark.flaky(reruns=5, reruns_delay=2)
```

Comment:
why is this new test already marked flaky, how can we make it not flaky? Retries add to extra runtime for test runs which is not viable. The retry should be a short term solution to unblock we need to long term remove retries from other tests as well.

Reply:
The flaky test was added to one of the existing tests. So I added that to the new test as well thinking that was a standard. I can remove that.